packages/web/src/pages/feed-page/FeedPageProvider.d.ts

```diff
@@ -0,0 +1,10 @@
+import { FeedPageContentProps } from './types'
```

Comment:
do these d.ts files just add top-level types for our old js files?

Reply:
exactly, the new upgrades complained about types that weren't being caught before cause i think older versions of react would just throw it's hands up and say `(props: any) => any` before, whereas now it is asking to actually type the js files

Reply:
Perfect! Love this change. Someday we'll be 100% typescript, but until that day, this is close enough